pkg/tls/certRenewer.go

```diff
@@ -321,7 +307,7 @@ func (c *CertRenewer) ValidCert() (bool, error) {
 	var managedByKyverno bool
 	snameTLS := GenerateTLSPairSecretName(certProps)
 	snameCA := GenerateRootCASecretName(certProps)
-	unstrSecret, err := c.client.GetResource("", "Secret", certProps.Namespace, snameTLS)
+	unstrSecret, err := c.client.CoreV1().Secrets(certProps.Namespace).Get(context.TODO(), snameTLS, metav1.GetOptions{})
```

Comment:
Should we update the var name now that it is typed?

Reply:
fixed